RELEASE_NOTES.rst

```diff
@@ -0,0 +1,29 @@
+coala-bears 0.1.0 alpha
+=======================
+
+coala-bears is a Python package containing all the bears that is used by coala.
+
+A bear is meant to do some analysis on source code. The source code will be
+provided by coala so the bear doesn’t have to care where it comes from or where
+it goes.
+
+We have a total of 51 bears.
+You can see all of them `here <https://gist.github.com/Adrianzatreanu/cf2d0c8b2ecd542a4860>`_.
+
+It has bears for powerful languages, such as:
+
+- Java
+- C++
+- HTML
+- CSS
+- even natural language!
+- and 25 more programming languages.
+See `this <https://gist.github.com/Adrianzatreanu/cf334ae9f19976e2d5d7>`_.
```

Comment:
I personally am in favour of including the full list, this are the release notes, people will only read them if they really wanna know what's going on. They are here because they clicked on some like on some tweet or whatever